src/main/java/frc/robot/trajectory/Vector.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.trajectory;
+
+public class Vector {
```

Comment:
Same as with Point, could probably be replaced with Translation2D or Pose. https://docs.wpilib.org/en/stable/docs/software/advanced-controls/geometry/pose.html